localeval.js

```diff
@@ -1,6 +1,10 @@
 // Thaddée Tyl <thaddee.tyl@gmail.com>. License: CC-BY v3.
 "use strict";
 var node_js = typeof exports === 'object';
+if ("undefined" !== typeof process && process.browser) {
+    // webpack sets browser property on process global
+    node_js = false;
+}
```

Comment:
I think `var node_js = typeof exports === 'object' && !(this.process !== undefined && process.browser === undefined);` works. What do you think?